code/modules/client/asset_cache.dm

```diff
@@ -358,16 +358,16 @@ var/list/asset_datums = list()
 
 /datum/asset/simple/faction_icons
 	assets = list(
-		"faction_EPMC.png" = 'icons/misc/factions/ECFlogo.png',
-		"faction_Zeng.png" = 'icons/misc/factions/ZhenHulogo.png',
-		"faction_Zavod.png" = 'icons/misc/factions/Zavodlogo.png',
-		"faction_NT.png" = 'icons/misc/factions/NanoTrasenlogo.png',
-		"faction_Idris.png" = 'icons/misc/factions/Idrislogo.png',
-		"faction_Hepht.png" = 'icons/misc/factions/Hephaestuslogo.png',
-		"faction_unaffiliated.png" = 'icons/misc/factions/Unaffiliatedlogo.png',
-		"faction_PMCG.png" = 'icons/misc/factions/pmcglogo.png',
-		"faction_Orion.png" = 'icons/misc/factions/Orionlogo.png',
-		"faction_SCC.png" = 'icons/misc/factions/SCClogo.png'
+		"faction_EPMC.png" = 'html/images/factions/ECFlogo.png',
+		"faction_Zeng.png" = 'html/images/factions/zenghulogo.png',
+		"faction_Zavod.png" = 'html/images/factions/zavodlogo.png',
+		"faction_NT.png" = 'html/images/factions/nanotrasenlogo.png',
+		"faction_Idris.png" = 'html/images/factions/idrislogo.png',
+		"faction_Hepht.png" = 'html/images/factions/hephlogo.png',
+		"faction_INDEP.png" = 'html/images/factions/unaffiliatedlogo.png',
```

Comment:
"INDEP" should be lowercase since it's not an abbreviation.

Reply:
no, will break /datum/faction/proc/get_logo_name()